csrc/quantization/quant_llm/quant_matmul.cuh

```diff
@@ -0,0 +1,354 @@
+//    Copyright 2024 FP6-LLM authors
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+//
+// This file is modified from
+// https://github.com/usyd-fsalab/fp6_llm/blob/5df6737cca32f604e957e3f63f03ccc2e4d1df0d/fp6_llm/csrc/include/kernel_matmul.cuh
+
+#include "configs.h"
+#include "utils/gmem.cuh"
+#include "utils/core.cuh"
+
+/************************** Bitwidth of Weight Segments
+ * ************************/
```

Comment:
Since these files seem to be copied, you may want to ignore them with clang-format to preserve their original formatting, see https://github.com/vllm-project/vllm/blob/cc4325b66ac49e403ed9e1a8c38156a5324e1174/format.sh#L238-L248